webhook.go

```diff
@@ -60,15 +80,29 @@ func (c *Client) GetWebhookWithContext(ctx context.Context, webhookID string) (o
 	return
 }
 
+// Deprecated: Use UpdateWebhook instead.
```

Comment:
Question: Have we confirmed the godoc pics up on this correctly or do we instead need to have this be the 2nd line in the summary?